src/renderer/pages/Profile/ProfileAll.vue

```diff
@@ -38,7 +38,7 @@
             </div>
 
             <span class="font-bold my-2 text-lg">
-              {{ profileBalance(profile) }}
+              {{ profileBalance(profile) | truncate(15) }}
```

Comment:
Could you add tests for this, please. An alternative place to truncate could be in the `profileBalance` method as that already has 1 test, just add additional tests for longer variations.

Reply:
Wouldn't the `truncate` css class a better alternative here? Truncating like above will yield unexpected results for locales where the currency symbol follows the amount.

Reply:
@alexbarnsley tests added.

Reply:
@brenopolanski I actually like @dated's suggestion to use the `truncate` css class 😅 Would you mind trying that approach instead please. Sorry, I only just saw dated's reply

Reply:
@alexbarnsley @dated i tested using `truncate` css class and the result is better:

![Screenshot from 2020-02-12 11-43-32](https://user-images.githubusercontent.com/1894191/74345551-03efdc00-4d8d-11ea-9429-03749ae62b1a.png)

For the `truncate` class works with `<span>` tag it's necessary to use the `display: inline-block` property, then the `.my-2` class works well now.